mmv1/templates/terraform/examples/sql_database_deletion_policy.tf.erb

```diff
@@ -0,0 +1,19 @@
+# [START cloud_sql_database_create]
+resource "google_sql_database" "<%= ctx[:primary_resource_id] %>" {
+  name     = "<%= ctx[:vars]['database_name'] %>"
+  instance = google_sql_database_instance.instance.name
+  deletion_policy = "ABANDON"
```

Comment:
Do these get cleaned up automatically when the parent instance is deleted? I don't want to be abandoning DBs in our test environment as that would get expensive!

Reply:
yes. The "database" here is a resource that exists the instance created by google_sql_database_instance. Destroying the instance remove the database from existence.